src/main/java/kg/jarkyn/cobspecserver/Response.java

```diff
@@ -0,0 +1,13 @@
+package kg.jarkyn.cobspecserver;
+
+public class Response {
+    private String status;
+
+    public Response(String status) {
+        this.status = status;
+    }
+
+    public byte[] getBytes() {
```

Comment:
This is more of a gut feeling right now, but it feels too much of a technical detail here to know about a byte array.